tools/pipelines/templates/include-telemetry-setup.yml

```diff
@@ -2,145 +2,84 @@
-            isTestBranch=$(isTestBranch)
-            toolAbsolutePath=$(toolAbsolutePath)
-            artifactPipeline=$(artifactPipeline)
-            artifactBuildId=$(artifactBuildId)
-            feed=$(feed)
-            devFeed=$(devFeed)
-            testWorkspace=${{ parameters.testWorkspace }}
-            loggerPackage=${{ parameters.loggerPackage }}
-
-          Build Params
-            SourceBranch=$(Build.SourceBranch)
-          "
-
-    - task: Bash@3
-      displayName: Create test directory
-      inputs:
-        targetType: 'inline'
-        script: |
-          mkdir ${{ parameters.testWorkspace }}
-
-    - task: Bash@3
-      displayName: Initialize npmrc
-      inputs:
-        targetType: 'inline'
-        workingDirectory: ${{ parameters.testWorkspace }}
-        # Note: $(ado-feeds-build) and $(ado-feeds-office) come from the ado-feeds variable group
-        script: |
-          echo Initialize package
-          npm init --yes
-
-          echo Generating .npmrc
-          echo "registry=https://registry.npmjs.org" >> ./.npmrc
-          echo "always-auth=false" >> ./.npmrc
-
-          echo "@fluidframework:registry=$(feed)" >> ./.npmrc
-          echo "@fluid-experimental:registry=$(feed)" >> ./.npmrc
-          echo "@fluid-tools:registry=$(feed)" >> ./.npmrc
-
-          # This is confusing, but we are using the .npmrc here to load dependencies from different feeds. These scopes
-          # must be loaded from the "dev" feed because that is the only place they are published. Ideally this logic
-          # will be centralized outside of CI in the future.
-          echo "@fluid-internal:registry=$(devFeed)" >> ./.npmrc
-          echo "@fluid-private:registry=$(devFeed)" >> ./.npmrc
-
-          # This scope must be loaded from the "build" feed
-          echo "@ff-internal:registry=$(ado-feeds-build)" >> ./.npmrc
-
-          # This scope must be loaded from the internal Office feed
-          echo "@microsoft:registry=$(ado-feeds-office)" >> ./.npmrc
-
-          echo "always-auth=true" >> ./.npmrc
-          cat .npmrc
-
-    # Auth to internal feed
-    - task: npmAuthenticate@0
-      displayName: 'npm authenticate (internal feed)'
-      inputs:
-        workingFile: ${{ parameters.testWorkspace }}/.npmrc
+- name: pathToTelemetryGenerator
+  type: string
+  default: '$(Build.SourcesDirectory)/tools/telemetry-generator'
 
-    # Install test and logger package
-    - task: Npm@1
-      displayName: 'npm install aria logger'
-      inputs:
-        command: 'custom'
-        workingDir: ${{ parameters.testWorkspace }}
-        customCommand: 'install ${{ parameters.loggerPackage }}'
-        customRegistry: 'useNpmrc'
+- name: devFeedUrl
+  type: string
 
-    - task: Bash@3
-      displayName: 'Prepare telemetry-generator'
-      inputs:
-        targetType: 'inline'
-        workingDirectory: $(toolAbsolutePath)
-        script: |
-          cp ${{ parameters.testWorkspace }}/.npmrc . ;
```

Comment:
Now the workingDirectory is where we setup .npmrc from the start so no need to copy it.